plugins/scaffolder/api-report.md

```diff
@@ -275,6 +281,12 @@ const scaffolderPlugin: BackstagePlugin<
 export { scaffolderPlugin as plugin };
 export { scaffolderPlugin };
 
+// Warning: (ae-forgotten-export) The symbol "TaskPageProps" needs to be exported by the entry point index.d.ts
```

Comment:
Also probably want to try and fix these warnings.

Some of the exported things will need  a `@public` tag, and some things will need some exporting, and some will need documentation. If you're not comfortable with the docs, you can leave it for now as we're going to address that soon, but let's get rid of these warnings.

Reply:
Ive added documentation and removed the warnings.